libs/utils/visualization_utils.py

```diff
@@ -358,6 +362,9 @@ def visualize_boxes_and_labels_on_image_array(
                     box_to_color_map[box] = STANDARD_COLORS[
                         classes[i] % len(STANDARD_COLORS)
                         ]
+                if face_labels:
+                    facemask_label = f"Facemask: {face_index[face_labels[i]]}"
```

Comment:
the label is just text?

Reply:
Yes, it's new a label bellow to the current `Pedestrian ..` showing the mask usage.
![Screenshot from 2020-11-16 10-13-29](https://user-images.githubusercontent.com/26575323/99256752-a04f7f00-27f4-11eb-8260-2026581c1b92.png)